src/main/java/uk/gov/hmcts/darts/common/exception/DartsApiException.java

```diff
@@ -59,4 +62,25 @@ public DartsApiException(DartsApiError error, String detail, Throwable throwable
         this.detail = detail;
     }
 
+    @Getter
+    @RequiredArgsConstructor
+    public enum DartsApiErrorCommon implements DartsApiError {
+        FEATURE_FLAG_NOT_ENABLED(
+            "FEATURE_FLAG_NOT_ENABLED",
+            HttpStatus.NOT_IMPLEMENTED,
+            AddAudioTitleErrors.USER_CANT_APPROVE_THEIR_OWN_DELETION.getValue()
```

Comment:
This doesn't look like the right thing to use here

Reply:
Good spot updated